src/CMakeLists.txt

```diff
@@ -109,6 +109,7 @@ int main() {
 }" HAVE_DXCORE_LIBRARY)
 if (HAVE_DXCORE_LIBRARY)
 	target_link_libraries(d3d12translationlayer dxcore)
+	target_link_options(d3d12translationlayer INTERFACE "/DELAYLOAD:dxcore.dll")

```

Comment:
Why do we want to delay load?

Reply:
I've heard feedback from people who want to use this library for uses other than mapping layers, but having to rely on DXCore makes it difficult, since that ties them to a particular OS version where it's present.